sklearn/model_selection/_search.py

```diff
@@ -1411,7 +1411,26 @@ class RandomizedSearchCV(BaseSearchCV):
         A generator over parameter settings, constructed from
         param_distributions.
 
+
+    Examples
+    --------
+    >>> from sklearn import linear_model, datasets
+    >>> from sklearn.model_selection import RandomizeSearchCV
+    >>> from scipy.stats import  uniform
+    >>> iris = datasets.load_iris()
+    >>> logistic = linear_model.LogisticRegression()
```

Comment:
May need to set parameters, `solver` and `multi_class`, in `LogisticRegression` to avoid warnings.